aplus/settings.py

```diff
@@ -126,6 +126,9 @@
 
     'js_jquery_toggle',
     'django_colortag',
+
+    # Debug Performance
+    'debug_toolbar',
```

Comment:
I don't think we want to enable this app in production, thus it must not be always included in `INSTALLED_APPS`. The end of this file already has if statements for modifying these settings. I suppose the app is only needed when `DEBUG = True`.